app/assets/javascripts/govuk_publishing_components/components/details.js

```diff
@@ -7,24 +7,29 @@ window.GOVUK.Modules.GovukDetails = window.GOVUKFrontend;
 (function (Modules) {
   function GemDetails ($module) {
     this.$module = $module
-    this.customTrackLabel = this.$module.getAttribute('data-track-label')
+    this.$summary = this.$module.querySelector('.govuk-details__summary')
+
+    this.customTrackLabel = this.$summary.getAttribute('data-track-label')
     this.detailsClick = this.$module.querySelector('[data-details-track-click]')
+    console.log(this.customTrackLabel, this.detailsClick)
```

Comment:
```suggestion
```

Reply:
Wait. So in removing one console.log, I managed to add three more? Wow I was not well yesterday afternoon. I am going to bed.

(fixed - hopefully)

Reply:
They were there - I just didn't highlight them, sorry!